src/Encamina.Enmarcha.Email.MailKit/EmailService.cs

```diff
@@ -156,7 +156,7 @@ public IEmailBuilder SetSubject(string subject)
         return this;
     }
 
-    private static SmtpClientOptions ValidateOptions(SmtpClientOptions smtpClientOptions)
+    private static SmtpClientOptions ValidateOptions(SmtpClientOptions? smtpClientOptions)
```

Comment:
It's a bit similar to the FileName case. This will generate an exception if that value is null...